simple-wp-helpdesk/src/Rest/BulkController.php

```
@@ -0,0 +1,183 @@
+	}
+
+	/**
+	 * Check that the current user can perform bulk ticket operations.
+	 *
+	 * @param WP_REST_Request $request REST request (unused — capability is global).
+	 * @return bool
+	 */
+	public function permissions_check( WP_REST_Request $request ): bool {
+		unset( $request );
+		return current_user_can( 'edit_posts' );
+	}
```

PHPMD (2.15.0)

[error] The method permissions_check is not named in camelCase. (undefined)

(CamelCaseMethodName)